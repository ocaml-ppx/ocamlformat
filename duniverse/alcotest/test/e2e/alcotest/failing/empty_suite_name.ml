let () =
  Alcotest.run ""
    [ ("alpha", [ Alcotest.test_case "1" `Quick (fun () -> assert false) ]) ]
