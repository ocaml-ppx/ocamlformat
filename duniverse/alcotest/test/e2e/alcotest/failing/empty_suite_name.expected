ERROR: Suite name cannot cannot be empty. Please pass a non-empty string to `run`.
